_posts/minutes/2016-04-22-22.md

```diff
@@ -0,0 +1,10 @@
+---
+    title: 'Minutes of Meeting on 22/04/2016'
+    author: qaisjp
+    category: minutes
+---
+This week's meeting covers details relating to sponsors.
+
+**Present**:
+
+Lisa Xie, Vesko Stefanov, Viktorija Lukosiute, Benedetta Mussati, Susanne Koster, Martynas Grigonis
```

Comment:
It seems that this meeting just noted down ideas for what sponsors could get for sponsoring CompSoc. I don't think it's the sort of thing we'd like sponsors to see in the future ("oh but these guys got that last year, why don't we get that?").

I made a post to log that a meeting happened, and to log who attended, but I guess we could just get rid of the post?